### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by gjf** [NekoX-Dev/NekoX#1028]
Just curious will this project move further or it is dead?
We have some info about future plans: https://github.com/NekoX-Dev/NekoX/issues/982 but they are stuck.
Also none of reported bugs are fixed.

I understand that every developer has own life but maybe users can obtain some information and decide will they wait (for smth) or move to another client?

**Comment by frknkrc44** [comment]
The developer said "I will definitely update nekox after submitting my paper by the end of September" so I think you can wait a few days for an update.

**Comment by dic1911** [comment]
@frknkrc44 禁止催促更新
No ETA on update

關鍵詞/Keywords:['ETA', 'update', '更新']

**Comment by luvletter2333** [comment]
Nope:) I'm sorry for the long delay, at least Oct 3 I will make up for all updates

**Comment by gjf** [comment]
> Nope:) I'm sorry for the long delay, at least Oct 3 I will make up for all updates

This is extremely good news!
I really love this client very much and it would be a pity if it was abandoned.

**Comment by luvletter2333** [comment]
> Just a note: NekoX is affected by https://www.cve.org/CVERecord?id=CVE-2023-5129. It should be very easy to fix it: you just need to update the submodule to https://github.com/webmproject/libwebp/commits/1.2.0.

Sure. I have noticed this issue.

**Comment by i0rt** [comment]
Last commit to dev 14 hours ago!! 🙌🙌 I love this client ❤️. Looking forward so much for it to be updated.

**Comment by amadicus** [comment]
It looks like the project is dead.  It's a pity(( 
R. I. P

**Comment by NikMAX2077** [comment]
> Nope:) I'm sorry for the long delay, at least Oct 3 I will make up for all updates

?

**Comment by amadicus** [comment]
> > Nope:) I'm sorry for the long delay, at least Oct 3 I will make up for all updates
> 
> ? 

What date is today??  What update did they promise?  Draw your own conclusions.

**Comment by albotroz** [comment]
looks as good as dead. No updates, despite security vulnerability.
3 October is also long gone.

Would advise against further use of this fork for security reasons.

**Comment by gjf** [comment]
Looks like we need to move to fork: https://github.com/nextalone/nagram

**Comment by et-ness** [comment]
> Looks like we need to move to fork: https://github.com/nextalone/nagram

I don't like nagram, it does not allow you to open issues and is not released on fdroid.

**Comment by linsui** [comment]
> As for fdroid it marks NekoX as dangerous because of non-fuxed vulnerability.

It's not since we have updated the libwebp with a patch.

**Comment by linsui** [comment]
Are you using the F-Droid version? If you are using the apk from upstream, F-Droid can't update it but still shows the warning. If you are using the F-Droid version then it should show you an update instead.

**Comment by gjf** [comment]
> Are you using the F-Droid version? If you are using the apk from upstream, F-Droid can't update it but still shows the warning. If you are using the F-Droid version then it should show you an update instead.

Unfortunately I am using upstream as I prefer GCM version for a number of reasons.

**Comment by et-ness** [comment]
> You can definitely open issues for NekoX - but nobody will fix them as development is dead )))

you were talking about nagram and I replied talking about nagram, what does nekox have to do with it? 
I know that nekox allows you to open issues, we are talking in an issue. With nagram this is not possible, is very bad, so I will never install nagram.

**Comment by gjf** [comment]
> > You can definitely open issues for NekoX - but nobody will fix them as development is dead )))
> 
> you were talking about nagram and I replied talking about nagram, what does nekox have to do with it? 
> I know that nekox allows you to open issues, we are talking in an issue. With nagram this is not possible, is very bad, so I will never install nagram.
> 

Mate, I am talking about possible alternatives those are as close as possible to this dead project.
Pros: everything is updated, the project is alive and the channel in Telegram published development together with Github.
The cons: you cannot publish issues.

Ok you can stay at NekoX if cons are more important for you than pros )))

**Comment by marek22k** [comment]
For me, because it's not in F-Droid.

**Comment by AlphaCraft9658** [comment]
It looks dead. Last release was 9 Months ago and last commit 8. Please, I want to use this client.

**Comment by linsui** [comment]
The last commit is 3 days ago. https://github.com/NekoX-Dev/NekoX/commits/dev Please stop spamming here, it doesn't help.

**Comment by AlphaCraft9658** [comment]
> The last commit is 3 days ago. https://github.com/NekoX-Dev/NekoX/commits/dev Please stop spamming here, it doesn't help.

Oh, sorry. Didn't see that. My mistake.

**Comment by amadicus** [comment]
> The last commit is 3 days ago. https://github.com/NekoX-Dev/NekoX/commits/dev Please stop spamming here, it doesn't help

Two months ago there was also activity, so what?

**Comment by y1111** [comment]
neko is releasing whats seems like beta versions atualized more frequently in this tg group https://t.me/nekox_broken

they call NekoX - ghetto editon . - Different package name, able to coexist with official NekoX (from pinned msg there)

by today last version posted is 10.6.1 ,posted january 23

**Comment by y1111** [comment]
https://github.com/PreviousAlone/Nnngram this project seems great too, it adds nekoX features and from other forks too , im experimenting . just dont like the translate that dont work like here on nekoX.
but that one is also not sending compiled releases for the github too . theyre posting on their tg too , here https://t.me/Nnngram
